### PR TITLE
CD-200604 Revamp TIFF to PNG arrayjoin code

### PR DESCRIPTION
**Pull request by revathi-murali**
## Reviewers

- [x] @priyankatapar 

## Summary of change

Revamped array join code when converting from multipage TIFF to PNG

### CODE REVIEW

File: vips.go [hunk 1]
```diff
@@ -302,42 +302,33 @@ func vipsRead(buf []byte) (*C.VipsImage, ImageType, error) {
...
+	frames := make([]*C.VipsImage, framesCount)
+
+	clear_frames := func() {
+		for i := 0; i < framesCount; i++ {
+			C.g_object_unref(C.gpointer(frames[i]))
 		}
-		// The interpretation for the incoming images is set as `sRGB` in imagica. Refer `parseColorspace` method in `params.go`.
-		// For a `sRGB` interpretation, bands should not be equal to 3. If so, then set an extra alpha band.
-		// Refer: https://github.com/libvips/libvips/issues/1525
-		bands := C.vips_image_get_bands(vipsImage)
```
Comment (priyankatapar) [inline vips.go:317]: We don't need to check for bands now?

Comment (revathi-murali) [reply vips.go:317]: Check is done inside C code. https://github.com/coupa/bimg/blob/2e1d0d01c19686f3d3c33a9009a8f281bf84f559/vips.h#L601-#L605

Comment (priyankatapar) [reply vips.go:317]: Okay, got it, I missed looking into this part ` if (!vips_image_hasalpha(*out))`.

**Comment by priyankatapar** [comment]
lgtm

**Comment by revathi-murali** [comment]
[lockit v4.9.3] merged via Lockit CLI on 2020-05-22 14:24:36.92574 +0530 IST m=+8.977859878

**Pull request merged**